src/test/javascript/spec/service/profile.service.spec.ts

```diff
@@ -0,0 +1,246 @@
+import { fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
+import { SERVER_API_URL } from 'app/app.constants';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { LocalStorageService, SessionStorageService } from 'ngx-webstorage';
+import { MockSyncStorage } from '../helpers/mocks/service/mock-sync-storage.service';
+import { MockRouter } from '../helpers/mocks/service/mock-route.service';
+import { Router } from '@angular/router';
+import { ProfileInfo } from 'app/shared/layouts/profiles/profile-info.model';
+import { FeatureToggle } from 'app/shared/feature-toggle/feature-toggle.service';
+import { ProgrammingLanguage, ProjectType } from 'app/entities/programming-exercise.model';
+
+describe('Logs Service', () => {
+    let service: ProfileService;
+    let httpMock: HttpTestingController;
+
+    const serverResponse = {
+        contact: 'artemis.in@tum.de',
+        imprint: 'https://ase.in.tum.de/lehrstuhl_1/component/content/article/179-imprint',
+        'guided-tour': {
+            courseShortName: 'artemistutorial',
+            tours: {
+                0: {
+                    cancel_tour: '',
+                },
+                1: {
+                    code_editor_tour: 'test',
+                },
+                2: {
+                    course_overview_tour: '',
+                },
+                3: {
+                    course_exercise_overview_tour: 'test',
+                },
+                4: {
+                    modeling_tour: 'UML Class Diagram',
+                },
+                5: {
+                    programming_exercise_fail_tour: 'test',
+                },
+                6: {
+                    programming_exercise_success_tour: 'test',
+                },
+                7: {
+                    tutor_assessment_tour: 'Patterns in Software Engineering',
+                },
+            },
+            //            "course-group-students": "artemis-artemistutorial-students"
```

Comment:
Should this code be removed? (Dead code)

Reply:
Yes, my mistake.